docs/src/administration/users.md

```diff
@@ -72,67 +60,27 @@ Available commands:
 * `platform user:role`
   * View or change a user's role
 
-For example, the following command would add `alice@example.com` with **Project Admin** role in the current project.
+ **Example**
+
+ Add `user1@example.com` to the project with Project Admin permissions:
 
 ```bash
-platform user:add alice@example.com -r admin
+platform user:add user1@example.com -r admin
 ```
+Once you add the user to the project, they receive an invitation email to confirm their details and a registration link.
+
+**Example**
 
-After adding `alice@example.com`, Alice will receive an invitation email asking to confirm her details and optionally, register for a Platform.sh account.
+To give User2 different levels of access depending on the environment type in the current project:
```

Comment:
I'd reword this to make the list that comes after more clear. Something like "Say you wanted to give User2 the following roles in different environment types:"